config/tables.yml

```diff
@@ -0,0 +1,2 @@
+phpbb.webpushnotifications.tables.notification_push: '%core.table_prefix%webpushnotifications_notification_push'
```

Comment:
would this be better?
`phpbb.wpn.tables.notification_push: '%core.table_prefix%wpn_notification_push`

Reply:
Or just `webpush`? What is more beautiful? :)